<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetCompra</class>
 <widget class="QWidget" name="WidgetCompra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabDevolucao">
      <attribute name="title">
       <string>Devoluções</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="WidgetCompraDevolucao" name="widgetDevolucao" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabResumo">
      <attribute name="title">
       <string>Resumo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="WidgetCompraResumo" name="widgetResumo" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPendentes">
      <attribute name="title">
       <string>Pendentes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="WidgetCompraPendentes" name="widgetPendentes" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabComprar">
      <attribute name="title">
       <string>Gerar Compra</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="WidgetCompraGerar" name="widgetGerar" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabConfirmar">
      <attribute name="title">
       <string>Confirmar Compra</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="WidgetCompraConfirmar" name="widgetConfirmar" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFaturamento">
      <attribute name="title">
       <string>Faturamento</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="WidgetCompraFaturar" name="widgetFaturar" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCompras">
      <attribute name="title">
       <string>Consumos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="WidgetCompraConsumos" name="widgetOC" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Histórico</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="WidgetHistoricoCompra" name="widgetHistorico" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetCompraGerar</class>
   <extends>QWidget</extends>
   <header>src/widgetcompragerar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCompraConfirmar</class>
   <extends>QWidget</extends>
   <header>src/widgetcompraconfirmar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCompraFaturar</class>
   <extends>QWidget</extends>
   <header>src/widgetcomprafaturar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCompraDevolucao</class>
   <extends>QWidget</extends>
   <header>src/widgetcompradevolucao.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCompraPendentes</class>
   <extends>QWidget</extends>
   <header>src/widgetcomprapendentes.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCompraResumo</class>
   <extends>QWidget</extends>
   <header>src/widgetcompraresumo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetHistoricoCompra</class>
   <extends>QWidget</extends>
   <header>src/widgethistoricocompra.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetCompraConsumos</class>
   <extends>QWidget</extends>
   <header>src/widgetcompraconsumos.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
